lib/tests/streamlit/web/bootstrap_test.py

```diff
@@ -309,6 +308,61 @@ def test_print_old_git_warning(self, mock_git_repo):
         out = sys.stdout.getvalue()
         self.assertIn("Streamlit requires Git 2.7.0 or later, but you have 1.2.3.", out)
 
+    @patch("streamlit.web.bootstrap.asyncio.get_running_loop", Mock())
+    @patch("streamlit.web.bootstrap.secrets.load_if_toml_exists", Mock())
+    @patch("streamlit.web.bootstrap._maybe_print_static_folder_warning")
+    def test_maybe_print_static_folder_warning_called_once_on_server_start(
+        self, mock_maybe_print_static_folder_warning
+    ):
+        """We should load secrets.toml on startup."""
+        bootstrap._on_server_start(Mock())
+        mock_maybe_print_static_folder_warning.assert_called_once()
+
+    @patch("os.path.isdir", Mock(return_value=False))
+    @patch("click.secho")
+    def test_maybe_print_static_folder_warning_if_folder_doesnt_exist(self, mock_echo):
+        """We should print a warning when static folder does not exist."""
+
+        mock_get_option = testutil.build_mock_config_get_option(
+            {"server.enableStaticServing": True}
+        )
+
+        with patch.object(config, "get_option", new=mock_get_option):
```

Comment:
No big deal, but we also have a more convenient way to do this:

```python
with testutil.patch_config_options({"server.enableStaticServing": True}):
  ...
```

(`build_mock_config_option` should possibly just be a private function in testutil)

Reply:
done